cloud/shared/bin/destroy.py

```diff
@@ -1,13 +1,14 @@
 """
 Destroy.py destroys Civiform deployment.
 """
+from typing import List
 
 from cloud.shared.bin.lib import terraform
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 from cloud.shared.bin.lib.setup_class_loader import get_config_specific_destroy
 
 
-def run(config: ConfigLoader):
+def run(config: ConfigLoader, params: List[str] = None):
```

Comment:
nit: you don't need to provide = None given that second parameter is always passed.